arbitrum-docs/node-running/reference/software-releases.md

```diff
@@ -0,0 +1,56 @@
+- Addition of the `MCOPY` EVM opcode introduced in [EIP-5656](https://eips.ethereum.org/EIPS/eip-5656) for cheaper memory copying.
+- Changes to the `SELFDESTRUCT` EVM opcode to reflect the behavior on L1 Ethereum, as outlined in [EIP-6780](https://eips.ethereum.org/EIPS/eip-6780).
+
+### Special notes on ArbOS 20: Atlas support for EIP-4844
+- Applications on Arbitrum will not have to be modified or take any explicit action to get the benefits of using EIP-4844 (i.e. the whole chain opts-in with ArbOS 20 “Atlas”).
+- ArbOS 20 “Atlas” adds support for Arbitrum chains to send data in a blob storage format to data availability layers, like L1 Ethereum, that support the blob transaction type. This includes Arbitrum One and Nova. ArbOS 20 “Atlas” does not add support for Arbitrum chains to receive data in a blob storage format. This means that any L3 Orbit chain settling to an L2 Arbitrum chain will post data to the underlying L2 Arbitrum chain as calldata. The L2 Arbitrum chain will then be able to post data to a L1 data availability layer like Ethereum using blobs.
+- There currently aren’t estimates on what the end-user gas savings of using blob data will be yet. This topic is something being actively worked on and monitored. Without mainnet data, the estimates for blob gas prices will not be accurate enough to reliably predict the cost reductions that users will experience - and even with mainnet data, the savings will vary by use case (i.e. no current way to predict the price impacts from all blob gas market participants yet). In general, however, the use of blobs will generally reduce the cost of using Arbitrum L2s.
+
+### Reference links
+- Original DAO proposal: [AIP: ArbOS Version 20 "Atlas"](https://forum.arbitrum.foundation/t/aip-arbos-version-20-atlas/20957)
+- [AIP: ArbOS Version 20 "Atlas" Snapshot Vote](https://snapshot.org/#/arbitrumfoundation.eth/proposal/0x813a366e287a872ada13d4f8348e771c7aa2d8c3cb00b2be31539ceab5627513)
+
+# ArbOS 11
+ArbOS 11 is shipped via Nitro v2.2.0, which is available on Docker hub with the image tag: `offchainlabs/nitro-node:v2.2.0-f7dc9de`. This release of Nitro is a mandatory release for Arbitrum One and Nova, pending governance approval. 
+
+### High-level description of ArbOS 11 changes
+- Addition of EVM changes made on L1 Ethereum as part of the [Shanghai upgrade](https://ethereum.org/history#shanghai), including support for the `PUSH0` opcode.
+- Improvements and fixes for [retryable tickets](../../arbos/l1-to-l2-messaging.mdx) to ensure that retryable fees used both the infrastructure fee account and the network fee account - just like how normal transactions do. This is important for [AnyTrust chains](../../inside-anytrust.mdx) like Arbitrum Nova where the infrastructure fee account pays out some fees to the Data Availability Committee (DAC) members but the network fee account does not.
```

Comment:
`Improvements and fixes for [retryable tickets](../../arbos/l1-to-l2-messaging.mdx) to ensure that retryable fees used both the infrastructure fee account and the network fee account - just like how normal transactions do.`

This sentence is a bit vague (at least for me), can we clarify it a bit?